### ISSUE TITLE
Reduced total lung capacity (TLC)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5596]
**Preferred term label:**

Reduced total lung capacity (TLC)

**Synonyms**

**Definition (free text, please give PubMed ID)**
-- see https://www.ncbi.nlm.nih.gov/books/NBK541029/

**Parent term (use hpo.jax.org/app)**
Parent term: Abnormal spirometry test HP:0032340

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

@mgriese, @katrinknoflach

**Comment by pnrobinson** [comment]
I am wondering if we really need an extra term or if we should revise the definition of this term
https://hpo.jax.org/app/browse/term/HP:0002091

Also, should we add more functional terms (see e.g. https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5299792/)

Def: An abnormal reduction of total lung capacity (TLC), which is the volume of air in the lungs upon the maximum effort of inspiration.  A restrictive ventilatory defect can be defined as as a TLC below the 5th percentile of normal predicted value with a normal FEV1/VC ratio.
PMID:28194273
Comment: The clinical indication for measuring lung volumes/capacities is usually when a patient presents with abnormal spirometry FEV1/FVC ratios significant for either obstructive or restrictive lung disease. Clinicians will then order additional testing either plethysmography, dilutional gas study, or radiographic imaging to confirm the presences of disease.

**Comment by pnrobinson** [comment]
On second thought it seems reasonable to have separate terms. 2091 is a composite inference that is not 100% synonymous.
